packages/elements-dev-portal/src/components/DevPortalProvider/index.tsx

```diff
@@ -13,7 +14,11 @@ const PlatformProvider: React.FC<DevPortalProviderProps> = ({
   platformAuthToken,
   children,
 }) => {
-  return <PlatformContext.Provider value={{ platformUrl, platformAuthToken }}>{children}</PlatformContext.Provider>;
+  return (
+    <PlatformContext.Provider value={{ platformUrl, platformAuthToken }}>
+      <MosaicProvider>{children}</MosaicProvider>
```

Comment:
Is this allowed to happen? Is there a better/more elegant way to get this in here so it's not exposed to the end user using dev portal?

Reply:
I think this is 💯 proper approach. My only worry would be conflicts in platform.

In platform MosaicProvider and DevPortalProvider are used in separation, which is probably ok, because mosaic is used explicitly there (or at least will be).

So what happens when there are two MosaicProviders in the code? Does it crash? Cause bugs? Or is it safe? 🤔 Those questions would have to be answered before we can make that change.

Reply:
Ahh okay that makes sense. Hmm I could try yalcing into platform to see what happens, but maybe even that wouldn't show the full story.

Reply:
Update: I yalced into ninja and interacted with the currently existing dev-portal components. I also added this Search component to test. Nothing misbehaved that I observed, though, I'm still not sure this is a completely reliable way to confirm that having two MosaicProviders is alright. Happy to give anything else a shot!

Reply:
Something else I noticed, is that when I try to implement the dev-portal `Search` component in ninja to make sure it doesn't get messed up with all the different providers, is that it's built to take strings of both `projectIds` and `workspaceId`. In ninja, I'm getting back `projectIds` and `workspaceIds` that are numbers and look nothing like those that we plug into the example. Even if I hard code that data, I don't receive any search results in platform, but I do in elements storybook and elements examples. We already have a search component that works and I am not trying to replace it, but if other people try to implement this in their environments, I'm worried it's not going to work for them, so I'd like to figure out why that's happening.

Reply:
I'm having more thoughts and questions about this whole search thing. Doesn't someone need to be able to grab their encoded `workspaceId` from their workspace to be able to plug it into the component since it uses `useGetNodes`? That means we need to have somewhere in the UI in platform that exposes this.